docs/en-US/component/cascader.md

```diff
@@ -142,96 +142,103 @@ cascader/panel
-| lazy          | whether to dynamic load child nodes, use with `lazyload` attribute                                         | boolean                 | —               | false      |
-| lazyLoad      | method for loading child nodes data, only works when `lazy` is true                                        | function(node, resolve) | —               | —          |
-| value         | specify which key of node object is used as the node's value                                               | string                  | —               | 'value'    |
-| label         | specify which key of node object is used as the node's label                                               | string                  | —               | 'label'    |
-| children      | specify which key of node object is used as the node's children                                            | string                  | —               | 'children' |
-| disabled      | specify which key of node object is used as the node's disabled                                            | string                  | —               | 'disabled' |
-| leaf          | specify which key of node object is used as the node's leaf field                                          | string                  | —               | 'leaf'     |
+## CascaderProps
+
+| Attribute      | Description                                                                                                | Type                                                | Default  |
+| -------------- | ---------------------------------------------------------------------------------------------------------- | --------------------------------------------------- | -------- |
+| expandTrigger  | trigger mode of expanding options                                                                          | ^[enum]`'click' \| 'hover'`                         | click    |
+| multiple       | whether multiple selection is enabled                                                                      | ^[boolean]                                          | false    |
+| checkStrictly  | whether checked state of a node not affects its parent and child nodes                                     | ^[boolean]                                          | false    |
+| emitPath       | when checked nodes change, whether to emit an array of node's path, if false, only emit the value of node. | ^[boolean]                                          | true     |
+| lazy           | whether to dynamic load child nodes, use with `lazyload` attribute                                         | ^[boolean]                                          | false    |
+| lazyLoad       | method for loading child nodes data, only works when `lazy` is true                                        | ^[Function]`(node: Node, resolve: Resolve) => void` | —        |
```

Comment:
**Suggestion:** maybe we should specify this `Resolve` type, this could be a little bit confusing.